### PR TITLE
feat: send same activeToken in 5 min for signin

### PR DESCRIPTION
**Pull request by howar31**
# Issue
send same activeToken in 5 min for signin

# Notice

# Dependency

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205413135009481

### CODE REVIEW

File: controllers/account.go [hunk 1]
```diff
@@ -83,13 +83,23 @@ func (mc *MembershipController) AuthByEmail(c *gin.Context, sendMailRoutePath st
...
 	if err == nil {
-		// update active token and token expire time
-		ra.ActivateToken = activeToken
-		ra.ActExpTime = time.Now().Add(time.Duration(15) * time.Minute)
-		if err = mc.Storage.UpdateReporterAccount(ra); err != nil {
-			return http.StatusInternalServerError, gin.H{"status": "error", "message": "Updating DB occurs error"}, err
+		// Calculate the time difference between ActExpTime and now
+		timeDifference := ra.ActExpTime.Sub(time.Now())
+
+		if timeDifference > 10*time.Minute {
```
Comment (Aylie-Chou) [inline controllers/account.go:92]: Question: is the expired time 10 minutes or 5 minutes ?

Comment (howar31) [reply controllers/account.go:92]: expired time is 5 minutes

Comment (Aylie-Chou) [reply controllers/account.go:92]: Got it, could we add a comment to idicate the spec that the activate token would not change in 5 minutes ?

Comment (howar31) [reply controllers/account.go:93]: done

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**